hashing/src/main/java/com/velocidi/apso/hashing/MurmurHash3.java

```diff
@@ -21,7 +21,7 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
 
-package com.velocidi.apso.hashing;
+package com.kevel.apso.hashing;
```

Comment:
Can we rollback the other changes in this file? I suspect some unwanted IDE shenanigans went unnoticed here.